js-pkg/gen-docs/src/main.ts

```diff
@@ -37,7 +37,7 @@ async function generateDocumentation() {
       path.join(jsPkgDir, 'react', 'src', 'main.tsx'),
     ]),
     generateDocumentationForProject(path.join(jsPkgDir, 'sdk'), [
-      path.join(jsPkgDir, 'sdk', 'src', 'main.ts'),
+      path.join(jsPkgDir, 'sdk', 'src', 'index.ts'),
```

Comment:
Is there a reason we're changing `main` to `index`? And just for this one package?

Reply:
Discussed offline, changed back.